great_tables/_utils_render_html.py

```diff
@@ -503,10 +515,30 @@ def create_body_component_h(data: GTData) -> str:
                 cell_styles = ""
 
             if is_stub_cell:
-                body_cells.append(f"""    <th class="gt_row gt_left gt_stub">{cell_str}</th>""")
+
```

Comment:
This is looking good, so just a small nit here 😅. Do you mind consolidating so there is a bit less duplicated code? Seems like the following could happen:

* change variables like `td_classes` and `th_classes` to have the same name. Similar with `th_classes_str`, etc..
* add a variable like el_name that is either "th" or "td"
* move all other logic (e.g. the appending of "gt_striped", the " ".join, the body_cells.append) out of the if / else, so it is only written once (e.g. move it to line 543 and unindent)
  - Use el_name in what should now be a single piece of code creating the final th or td string.

It looks like if something is a stub, that only affects the element name and some of the classes to be set. (But other logic is being included)

Reply:
Thanks, will make these changes!